lib/DBIx/Class/Schema/Loader/DBI/Oracle.pm

```diff
@@ -415,12 +415,20 @@ sub _dbh_column_info {
 
 sub _view_definition {
     my ($self, $view) = @_;
-
-    return scalar $self->schema->storage->dbh->selectrow_array(<<'EOF', {}, $view->schema, $view->name);
+    my $viewdef = scalar $self->schema->storage->dbh
+        ->selectrow_array(<<'EOF', {}, $view->schema, $view->name);
 SELECT text
 FROM all_views
 WHERE owner = ? AND view_name = ?
```

Comment:
I would flip the order of these WHERE clauses so that the parameter lists for both queries are in the same order.